.ci/azure-pipelines/build.yaml

```diff
@@ -0,0 +1,44 @@
+resources:
+  containers:
+    - container: fmt
+      image: pointcloudlibrary/fmt
+    - container: env1604
+      image: pointcloudlibrary/env:16.04
+    - container: env1910
+      image: pointcloudlibrary/env:19.10
+    - container: doc
+      image: pointcloudlibrary/doc
+
+pool:
```

Comment:
Seems unneeded since the pool is in each "task"

Reply:
I didn't quite get it, do you mean tasks predefined by [Azure](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#task)?

Reply:
The pool isn't needed since each `format`, `windows`, `mac`, `documentation`, etc. have their own pools.

Reply:
We could instead remove the pool from the individual jobs and define it globally, so that all jobs will use the default pool until specified otherwise like in `windows` and `mac`.
Having it global will be beneficial in case we want to update the pool version, them we have to do it only one place since `documentatio`', `format`, etc. share same pool.

Reply:
IMO, this adds a hidden dependency

Reply:
> adds a hidden dependency

True, didn't consider that.
Updated it in 82d25e1. "Explicit is better than implicit"